extensions/amp-base-carousel/0.1/test-e2e/helpers.js

```diff
@@ -78,3 +78,7 @@ export function getPrevArrow(controller) {
 export function getNextArrow(controller) {
   return controller.findElement(NEXT_ARROW_SELECTOR);
 }
+
+export function sleep(ms) {
```

Comment:
Note that other amp-base-carousel e2e tests import directly from `sleep-promise`:
https://github.com/ampproject/amphtml/blob/20f13046b968244f1eb4ebc1162f79e4f86250d1/extensions/amp-base-carousel/0.1/test-e2e/test-advance.js#L18

Reply:
Yeah, this was from my nit. I personally don't like the idea of importing an npm package for what is essentially a one-liner:
```js
const sleep = ms => new Promise(res => setTimeout(res, ms));
```

Reply:
Oh sorry I missed it! Thanks for clarifying here, makes sense to me. Let's file an issue to do a bulk replace?

Reply:
Removed sleep npm dependency for all e2e tests. Would it be prudent to create a presubmit check that guards against the use of npm packages?

Reply:
Don't think we should guard against all npm dependencies since they are often useful. We just need to weigh pros/cons for each dep. In this specific case, implementing ourselves is so trivial that I think the potential issues that arise from an external dep aren't worth it.

Thank you for the incredible followthrough here!